pkgs/servers/traefik/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildGoPackage, fetchurl, bash, go-bindata}:
+
+buildGoPackage rec {
+  name = "traefik-${version}";
+  version = "v1.3.8";
+
+  goPackagePath = "github.com/containous/traefik";
+
+  src = fetchurl {
+    url = "https://github.com/containous/traefik/releases/download/${version}/traefik-${version}.src.tar.gz";
+    sha256 = "6fce36dd30bb5ae5f91e69f2950f22fe7a74b920e80c6b441a0721122f6a6174";
+  };
+
+  buildInputs = [ go-bindata ];
+  sourceRoot = ".";
+  postUnpack = ''
+  files=`ls`
```

Comment:
Not sure if there is a nix way to do this but you probably at least want `ls -A` here to also get hidden files too. If find is available here (I'm not sure if it is?), then it may be more correct or robust to use it instead:

``` shell
find . -maxdepth 1 -not -path . -not path ./traefik -exec mv {} traefik \;
```

Otherwise if still using `mv`, it may be safer (in case of any 'unexpected' `ls` output, not sure) to use the `-t` option:

``` shell
mv -t traefik $files
```